src/components/SeButton.jsx

```diff
@@ -33,7 +33,7 @@ const SeButton = (props) => {
       <Button
         className={cssClasses}
         text={isLoading ? buttonLoadingText() : text}
-        onClick={onClick}
+        onClick={!isDisabled ? onClick : ""}
```

Comment:
this wouldn't hurt anything, but why wouldn't you use 

`disabled={isDisabled}`

?

Reply:
That button is the component library button. Do we have a disabled property for buttons? Looked to me like the library just passed a css class to handle disabled but maybe Im wrong.

Reply:
so yes Im special and missed that. thanks